sway-ir/src/printer.rs

```diff
@@ -173,6 +173,21 @@ pub fn function_print(context: &Context, function: Function) {
     );
 }
 
+/// Print an instruction to stdout.
+pub fn instruction_print(context: &Context, ins_value: &Value) {
```

Comment:
this function does not seem to be used; but im assuming its a good helper to have?

Reply:
> this function does not seem to be used; but im assuming its a good helper to have?

yep. eases debugging.